kie-wb-common-react/boxed-expression-editor/server.js

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path');
+const express = require('express');
+const app = express();
+const port = 8080;
+
+app.use(express.static(path.join(__dirname, '/dist')));
+app.get('*', (req,res) =>{
+  res.sendFile(path.join(__dirname + '/dist/index.html'));
+});
+app.listen(port, () => console.log(`Listening on port ${port}`));
```

Comment:
If we remove `express`, we'll be probably able to drop this whole file.

Reply:
Yes, we will remove `express`, like stated in https://github.com/vpellegrino/kie-wb-common/pull/3#discussion_r522795145, and so this server too.